apps/nowcasting-app/components/charts/gsp-pv-remix-chart/index.tsx

```diff
@@ -33,6 +38,19 @@ const GspPvRemixChart: FC<{
     gspForecastData?.find((fc) => formatISODateString(fc?.targetTime) === selectedTime) ||
     ({} as any);
   const pvPercentage = (forcastAtSelectedTime.expectedPowerGenerationNormalized || 0) * 100;
+
+  // set ymax to the installed capacity of the graph
+  var yMax = gspInfo?.installedCapacityMw;
+
+  // lets round it up to the 'yMax_levels' so that the y major ticks look right.
+  for (var i = 0; i < yMax_levels.length; i++) {
+    var level = yMax_levels[i];
+    yMax = yMax! < level ? level : yMax;
```

Comment:
this is perfectly good use of "!" operator and it's always better to not depend on it possible , instead it can be something like
```
 if(!gspInfo) return <div> gsp info not found </div>;
or 
  var yMax = gspInfo?.installedCapacityMw || 100;
```

Reply:
Thanks